content/4_Gold/Intro_DP.mdx

```diff
@@ -47,6 +47,15 @@ export const problems = {
 			['DP'],
 			'dp[i][j] -> minimum sum of net sizes needed to catch m snakes with k changes, precompute maximums'
 		),
+		new Problem(
+			'Gold',
+			'Angry Cows',
+			'597',
+			'Easy',
+			false,
+			['DP', '2P', 'Binary Search'],
```

Comment:
Should we keep or remove from Binary Search module?

Reply:
Good catch.... I don't know the problem -- if it involves dp why is it in the binsearch module, and why is it starred there?

Reply:
It's bsearch on ans + DP to check.